vue/src/components/SettingsPanel.vue

```diff
@@ -215,6 +224,21 @@ watch(hiddenCanvas, () => {
         >
       </label>
     </div>
+    <div class="form-control">
+      <label class="label cursor-pointer">
+        <img
+          ref="polygonImg"
```

Comment:
`polygonImg` is a ref to two locations (once for the region polygon and once for the site outline).  It means that only one of them is set as an actual image.  The site Outline is just an empty image of a specific size.  Not a big deal if we want to indicate site-outline to be an empty image and take advantage of the default border for images.  I just suggest we remove the reference from it or specify a different reference like the previous one `siteOutlineImg` if we want to tie it to a specific color.  These are simpler than the patterns so they could just be 'div' of size X with computed style or class indicating the border color as well.
![image](https://user-images.githubusercontent.com/61746913/225979715-8cc9eeda-45df-459f-98d3-6eade42a4bff.png)

Reply:
I updated it to just use a `div` with a grey border for the icon instead of an img/ref approach https://github.com/ResonantGeoData/RD-WATCH/pull/159/commits/a7443e86375f4fd82b60990d7ebbd988b070f0d6.